src/static/app/login/login.scss

```diff
@@ -75,4 +79,105 @@
 } 
 
 
+.login {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  background-color: #f0f2f5;
+
+  .wrapper {
+    max-width: 400px;
+    background-color: #ffffff;
+    border-radius: 10px;
+    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1), 0 1px 3px rgba(0, 0, 0, 0.08);
+  }
+
+  #formContent {
+    padding: 30px;
+    text-align: center;
+
+    .profile-pic {
+      width: 100px;
+      height: 100px;
+      border-radius: 50%;
+      overflow: hidden;
+      margin: 0 auto;
+    }
+
+    img {
+      max-width: 100%;
+    }
+
+    label {
+      display: block;
+      font-size: 14px;
+      font-weight: 600;
+      color: #4a4a4a;
+      margin-bottom: 10px;
+    }
+
+    input {
+      width: 100%;
+      padding: 10px;
+      font-size: 14px;
+      border: 1px solid #e0e0e0;
+      border-radius: 5px;
+      outline: none;
+      transition: border-color 0.3s;
+      margin-bottom: 20px;
+
+      &:focus {
+        border-color: #007bff;
+      }
+    }
+
+    .check-lg {
+      margin-bottom: 20px;
+    }
+
+    button {
+      width: 100%;
+      padding: 10px;
+      font-size: 16px;
+      font-weight: 600;
+      border: none;
+      border-radius: 5px;
+      cursor: pointer;
+      outline: none;
+      transition: background-color 0.3s;
+
+      &:not(:last-child) {
+        margin-bottom: 10px;
+      }
+    }
+
+    .blue-button {
+      background-color: #007bff;
+
+      &:hover {
+        background-color: #0056b3;
+      }
+    }
```

Comment:
So for our blue colors we use the Hatian flag blue (commented on top of this file) - #B30D2C

The app colors matches P4h logo and Haitian flag (red, white, and blue).